src/v2i-hub/PedestrianPlugin/src/PedestrianPlugin.cpp

```diff
@@ -204,7 +204,7 @@ void PedestrianPlugin::UpdateConfigSettings()
     }
 	else
 	{
-		PLOG(logWARNING) << "Invalid configured data provider. Pedestrian Plugin requires valid data provider (FLIR, PSM)!";
+		PLOG(logWARNING) << "Invalid configured data provider. Pedestrian Plugin requires valid data provider (FLIR, PSM, SDSM)!";
```

Comment:
Both PSM and SDSM are not valid dataprovider options in your logic

Reply:
Caught and updated. Further updating to change to REST